core/src/main/scala/org/apache/spark/metrics/MetricsSystem.scala

```diff
@@ -235,3 +235,32 @@ private[spark] object MetricsSystem {
     new MetricsSystem(instance, conf, securityMgr)
   }
 }
+
+private[spark] object MetricsSystemInstances {
+  // The Spark standalone master process
+  val MASTER = "master"
```

Comment:
Maybe you lost `core/src/main/scala/org/apache/spark/deploy/master/MasterSource.scala`
MasterSource have the code:
`override val sourceName = "master"`
There could reference `MetricsSystemInstances.MASTER`.

Reply:
Thanks. The locations you pointed out are metrics `Source `names, which should not reference `MetricsSystem `instance names. Of course your comment is enlightening. Maybe we could also put metrics `Source `names together ?

Reply:
I'm sorry.`MetricsSystemInstances` is used for name of MetricsSystem instance.